tools/d2g/d2gtest/schemas/test_suites.yml

```diff
@@ -62,6 +60,12 @@ properties:
               description: |-
                 Detailed information about what the test case tests
               type: string
+            isTaskDef:
+              title: Is Task Definition
+              description: |-
+                Set to true if the test input are task definitions in
+                dockerWorkerTaskDefinition and genericWorkerTaskDefinition.
+              type: boolean
```

Comment:
I don't think this is really needed, since you can see if a payload or full task definition has been included.